README.md

```diff
@@ -17,12 +17,28 @@ Please view the footnotes before you get started.
 2. [Python 3](https://www.python.org/)
 
 ## Examples
-Your phone should be connected to your computer with a USB cable and USB Debugging should be turned on. Good Lock should be open and opened up to the custom color picker screen with the tile you want to change set as the selected tile.   
-You should run this command for each tile you want to change.
+Your phone should be connected to your computer with a USB cable and USB Debugging should be turned on. Good Lock should be open and opened up to the **custom color picker screen** with the tile you want to change set as the selected tile.
 
-`python hex2goodlock.py "#253f3f"`  
-`python hex2goodlock.py "#253237"`
+If you do not have a button at the bottom of the color picker screen that says "Custom", update Good Lock.
 
+You should run this command for each tile you want to change, replacing the hexadecimal color code as needed:
+
+`python hex2goodlock.py "#253f3f"`
+
+### Recording
+
+<p align="center">
+<img src="https://github.com/andrewpe/goodlock-hex2color/raw/master/images/recording.gif" />
```

Comment:
This file will exist after merge; have to link to it this way to center it in Markdown